builtin/settingtypes.txt

```diff
@@ -602,6 +602,10 @@ lighting_boost_spread (Light curve mid boost spread) float 0.2 0.0 1.0
 texture_path (Texture path) path
 
 #    The rendering back-end for Irrlicht.
+#    A restart is required after changing this.
+#    NB: on Android, stick with OGLES1 if unsure! App may fail to start otherwise.
```

Comment:
What's NB?

Reply:
[Nota bene](https://en.wikipedia.org/wiki/Nota_bene)

Reply:
'Note' is better.